code/parachain/frame/ibc/Cargo.toml

```diff
@@ -143,6 +143,7 @@ runtime-benchmarks = [
   "tendermint",
   "hex-literal",
   "pallet-timestamp",
+  "pallet-timestamp/runtime-benchmarks",
```

Comment:
both of these aren't required, `pallet-timestamp/runtime-benchmarks` brings in `pallet-timestamp`